docs/admin/access.rst

```diff
@@ -315,53 +302,49 @@ The authentication models consist of several objects:
 
 A user has to be a member of a group linked to the project, or any component
 inside that project. Having membership is enough, no specific permissions are
 needed to browse the project (this is used in the default `Viewers` group, see
 :ref:`default-groups`).
 
-Access for browse to a component
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Component browsing access
+^^^^^^^^^^^^^^^^^^^^^^^^^
 
-A user can access unrestricted components once able to access the components’
-project (and will have all the permissions the user was granted for the
-project). With :ref:`component-restricted` turned on, access to the component
-requires explicit permissions for the component (or a component list the component is in).
+A user can access all components with the same permissions of
+projects the user is granted access to.
+With :ref:`component-restricted` on, access to components
+or component lists are granted explicitly.
 
 .. _perm-check:
 
-Scope of groups
-^^^^^^^^^^^^^^^
+Group permissions
+^^^^^^^^^^^^^^^^^
 
-The scope of the permission assigned by the roles in the groups are applied by
-the following rules:
+Roles in the groups are made from the following rules:
 
-- If the group specifies any :guilabel:`Component list`, all the permissions given to
-  members of that group are granted for all the components in the component
-  lists attached to the group, and an access with no additional permissions is
-  granted for all the projects these components are in. :guilabel:`Components`
+- Specifying any :guilabel:`Component list` for a group grants members in it
+  permission for all included components, along with browsing access
+  to the projects they are in. :guilabel:`Components`
   and :guilabel:`Projects` are ignored.
 
-- If the group specifies any :guilabel:`Components`, all the permissions given to
-  the members of that group are granted for all the components attached to the
-  group, and an access with no additional permissions is granted for all the
-  projects these components are in. :guilabel:`Projects` are ignored.
+- Specifying any :guilabel:`Components` for the group grants members of it
+  permission for all included components, along with browsing access
+  to the projects they are in. :guilabel:`Projects` are ignored.
 
-- Otherwise, if the group specifies any :guilabel:`Projects`, either by directly
-  listing them or by having :guilabel:`Projects selection` set to a value like :guilabel:`All
-  public projects`, all those permissions are applied to all the projects, which
-  effectively grants the same permissions to access all projects
-  :ref:`unrestricted components <component-restricted>`.
+- Otherwise, adding any :guilabel:`Projects` to a group (either by directly
+  listing them or by having :guilabel:`Selected projects` set to :guilabel:`All
+  public`) grants the group permissions for the projects. (Effectively
+  the same permissions as being granted access to :ref:`unrestricted components <component-restricted>`
```

Comment:
Didn't look into what "which effectively grants the same permissions to access all projects" from the merge error means. Otherwise rewritten from what it was.